dashboard/app/models/ability.rb

```diff
@@ -30,7 +30,14 @@ def initialize(user)
         District,
         Workshop,
         Cohort,
-        WorkshopAttendance
+        WorkshopAttendance,
+        # PLC Stuff
+        ProfessionalLearningCourse,
+        ProfessionalLearningModule,
+        ProfessionalLearningTask,
+        PlcEvaluationQuestion,
+        PlcEvaluationAnswer,
+        UserProfessionalLearningCourseEnrollment
```

Comment:
It might be easier to do the namespacing now?  `Plc::Module`, `Plc::EvaluationAnswer`, etc. seems way more readable.  I was having trouble above knowing which controller I was in b/c the names are so long =/